crates/wasi-nn/Cargo.toml

```diff
@@ -24,8 +24,20 @@ wasmtime = { workspace = true, features = ["component-model"] }
 
 # These dependencies are necessary for the wasi-nn implementation:
 tracing = { workspace = true }
-openvino = { version = "0.6.0", features = ["runtime-linking"] }
 thiserror = { workspace = true }
+openvino = { version = "0.6.0", features = [
+    "runtime-linking",
+], optional = true }
+
+[target.'cfg(windows)'.dependencies.windows]
+version = "0.52"
+features = [
+    "AI_MachineLearning",
+    "Storage_Streams",
+    "Foundation",
+    "Foundation_Collections",
```

Comment:
Are all of these features needed? I would have assumed that only `"AI_MachineLearning"` is needed, which adds the right dependent features (see [here](https://github.com/microsoft/windows-rs/blob/a2fbd102010a20ac35ff0d950e44f193cd032751/crates/libs/windows/Cargo.toml#L36)). The reason I'm thinking about this is that the `windows` crate totals 11MB and I don't want to unnecessarily bloat Wasmtime binaries on Windows.

Reply:
"Storage_Streams" and "Foundation_Collections" are needed because some WinML APIs need data structures defined by these two features. "Foundation" is removed.